frontend/package.json

```diff
@@ -51,10 +53,19 @@
     "gulp-strip-debug": "^3.0.0",
```

Comment:
Do we still need the `gulp` dependencies?

Reply:
nice catch, I'll remove those

Reply:
💖